x/mongo/driver/session/client_session.go

```diff
@@ -116,16 +118,15 @@ type Client struct {
 
 	// options for the current transaction
 	// most recently set by transactionopt
-	CurrentRc  *readconcern.ReadConcern
-	CurrentRp  *readpref.ReadPref
-	CurrentWc  *writeconcern.WriteConcern
-	CurrentMct *time.Duration
+	CurrentRc       *readconcern.ReadConcern
+	CurrentRp       *readpref.ReadPref
+	CurrentWc       *writeconcern.WriteConcern
+	CurrentWTimeout time.Duration
```

Comment:
See PR description concerning write concerns to learn why we need to maintain state a write concern timeout state here.